BossMod/Config/ConfigUI.cs

```diff
@@ -52,11 +65,80 @@ public ConfigUI(ConfigRoot config, WorldState ws)
             }
 
             SortByOrder(_roots);
+
+            _exSheet = Service.DataManager.GetExcelSheet<ExVersion>()!;
+            _cfcSheet = Service.DataManager.GetExcelSheet<ContentFinderCondition>()!;
+            _modules = ModuleRegistry.RegisteredModules
+                .Where(x => !x.Value.IsUncatalogued)
+                .OrderBy(x => x.Value.ExVersion)
+                .ThenBy(x => _cfcSheet.GetRow(x.Value.CFCID)?.ClassJobLevelSync)
+                .ThenBy(x => _cfcSheet.GetRow(x.Value.CFCID)?.ItemLevelRequired)
+                .ThenBy(x => _cfcSheet.GetRow(x.Value.CFCID)?.SortKey)
+                .ToDictionary(x => x.Key, x => x.Value);
+            _uncatalogued = _modules.Where(x => x.Value.IsUncatalogued || x.Value.ExVersion == 69).Select(x => x).ToDictionary(x => x.Key, x => x.Value);
+            _expacs = _modules.Where(x => x.Value.ExVersion != 69).Select(x => x.Value.ExVersion).Distinct().ToList()!;
```

Comment:
I think this really belongs to some other class, not config - config should still draw configuration...

Reply:
moved it all to the registry where it makes the most sense at least. Only thing further would be to make a catalogue specific class but that just seems unnecessary